universal-login-react/src/ui/Settings/ManageDevices.tsx

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
+import {DeviceInfo} from '@universal-login/commons';
 
-export const ManageDevices = () => (
+interface ManageDevicesProps {
+  devices?: DeviceInfo[];
+}
+
+export const ManageDevices = ({devices}: ManageDevicesProps) => (
   <div>
-    <button className="settings-btn">Remove this device</button>
-  </div>
+    {
+      devices && devices.map(({os}: DeviceInfo, index: number) => (
```

Comment:
I generally prefer default values for parameters and avoiding patterns like `x && x.map(...)`

Reply:
Replaced by conditional rendering info that devices are loading